packages/api-sdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metriport/api-sdk",
-  "version": "6.0.6",
+  "version": "6.1.0",
```

Comment:
I made a mistake to use `minor` instead of `prerelease` on the `publish:alpha` script, so it's a prod release, yes, but at least `latest` its not pointing to it:

<img width="781" alt="image" src="https://github.com/metriport/metriport/assets/2132564/d3128534-1357-4a50-b374-6a3aa1f26c35">

I'll update the npm script to include `prerelease` with `publish:alpha` to avoid this.